alterconfigs.go

```diff
@@ -63,10 +63,10 @@ type AlterConfigsResponseResource struct {
 // AlterConfigs sends a config altering request to a kafka broker and returns the
 // response.
 func (c *Client) AlterConfigs(ctx context.Context, req *AlterConfigsRequest) (*AlterConfigsResponse, error) {
-	resources := make([]alterconfigs.RequestResources, len(req.Resources))
+	resources := make([]alterconfigs.RequestResources, 0, len(req.Resources))
```

Comment:
Sorry I should have been more clear:

The previous commit was fine, I was just trying to explain why I requested the change, this latest commit will now error because we're setting the length to 0 so when we try to set `resources[i]` it will get an index out of bounds

Same with all the other changes to `make` in this commit

Reply:
fixed.